docs/distribution/README.md

```diff
@@ -0,0 +1,927 @@
+  * This has been a [desired item for OCI Artifacts][oci-artifacts-index] to support other artifact types which would base on Index.
+* An additional role check is performed, based on the artifact type. Also noted as a pro as registry operators may want to utilize this for other artifact types, making it a consistent model.
+
+> **Note:** this is our working/preferred method: See OCI image-spec issue: [Add Index Support for Artifact Type #806](https://github.com/opencontainers/image-spec/issues/806)
+
+### Signature Persistence - Option 2a: oci-index signing a multi-arch manifest
+
+Takin the above scenario further, a signature can be associated with an individual manifest, or a signature can be applied to an index. The index could be a multi-arch index (windows & linux), or the index might represent a [CNAB][cnab].
+
+In the below case, the `net-monitor` software is available as windows (`net-monitor:v1-win`) and linux (`net-monitor:v1-lin`) images, as well as a multi-arch index (`net-monitor:v1`)
+The platform specific images, and the multi-arch index are all signed by **wabbit-networks** and **acme-rockets**.
+
+<img src="../../media/signature-as-index-signing-multi-arch-index.png" width=1100>
+
+
+### Signature Persistence - Option 3: oci-manifest linked through oci-index
+
+This model is a hybrid of the 1 & 2, but moves the Persistence of the signature from the config object to a layer of an additional manifest.
+
+<img src="../../media/signature-as-manifest-via-index.png" width=950>
+
+1. **manifest digest for the `net-monitor:v1` image:** `sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m"`
+
+    ```JSON
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.manifest.v2+json",
+      "config": {
+        "mediaType": "application/vnd.oci.image.config.v1+json",
+        "digest": "sha256:111ca3788f3464fd9a06386c4d7a8e3018b525278ac4b9da872943d4cfea111c",
+        "size": 1906
+      },
+      "layers": [
+        {
+          "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+          "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+          "size": 32654
+        },
+        {
+          "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+          "digest": "sha256:ec4b8955958665577945c89419d1af06b5f7636b4ac3da7f12184802ad867736",
+          "size": 73109
+        }
+      ]
+    }
+    ```
+
+2. **index digest for the wabbit-networks signature** `sha256:222ibbf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cb222i`
+
+    ```json
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.index.v2+json",
+      "config": {
+        "mediaType": "application/vnd.cncf.notary.index.config.v2+jwt",
+        "digest": "sha256:222cb130c152895905abe66279dd9feaa68091ba55619f5b900f2ebed38b222c",
+        "size": 1906
+      },
```

Comment:
What's this object in the example here?

Reply:
Updated to say: the `net-monitor:v1` image